Styling the chat

kingo2157 opened the pull request.

kingo2157 committed:
Added Styling, dummy div to auto scroll

IRHM reviewed (changes requested):
Can you remove the css files and add them to the gitignore. Also remove your settings.json.. that should probably be in your user settings not in the workspace.

Remove main.dev.js as well.

kingo2157 committed:
innerHTML changed to safer method

IRHM committed:
Remove unnecessary files

IRHM committed:
Add some basic styling to make messages scrollable on its own

also fix insertAdjacentHTML inserting before div and scroll to bottom of div with a better method

IRHM commented:
I added some fixes to your code. Also please don't compile the JS, or if you do make sure you don't commit it by accident.

kingo2157 reviewed (commented):
These are changes that I have viewed; what does this do?

kingo2157 committed:
Merge branch 'main' of https://github.com/IRHM/chat-app-test into IRHM-main

kingo2157 committed:
Merge branch 'IRHM-main' into Styling

IRHM commented:
> These are changes that I have viewed; what does this do?

If you're talking about what I think you are, nothing. It's just a way to organize which changed files you have looked at.

---

Are you done styling? If not, keep the pr as a draft.
